e2etests/pkg/routes/routes.go

```diff
@@ -4,7 +4,6 @@ package routes
 
 import (
```

Comment:
About the commit message:
is there a them for the commit? like <word>: word before : can be E2E or API, or Controller ?
maybe `E2E: description`

note: I did not understood that flag means the return bool value, is it valid term?

Reply:
this done